core/test/audits/dobetterweb/uses-http2-test.js

```diff
@@ -5,26 +5,62 @@
  */
 
 import UsesHTTP2Audit from '../../../audits/dobetterweb/uses-http2.js';
-import {NetworkRecords} from '../../../computed/network-records.js';
 import {networkRecordsToDevtoolsLog} from '../../network-records-to-devtools-log.js';
-import {getURLArtifactFromDevtoolsLog, readJson} from '../../test-utils.js';
+import {createTestTrace} from '../../create-test-trace.js';
 
-const trace = readJson('../../fixtures/traces/progressive-app-m60.json', import.meta);
-const devtoolsLog = readJson('../../fixtures/traces/progressive-app-m60.devtools.log.json', import.meta);
+const baseResources = [{
```

Comment:
I think we can `baseResources` and just inline these beginning 3 requests everywhere they are used. The test file will be longer but it gives each test more control over what it's testing.